Classes/Authentication/OpenIdConnectProvider.php

```diff
@@ -207,8 +207,12 @@ private function getConfiguredRoles(IdentityToken $identityToken): array
         }
 
         if (isset($this->options['rolesFromClaims']) && is_array($this->options['rolesFromClaims'])) {
-            $claims = array_unique($this->options['rolesFromClaims']);
-            foreach ($claims as $claim) {
+            foreach ($this->options['rolesFromClaims'] as $claim) {
+                $mapping = null;
+                if (is_array($claim)) {
+                    $mapping = $claim['mapping'] ?? null;
+                    $claim = $claim['name'];
```

Comment:
What if `$claim['name']` is not set?

Reply:
Hah. You found my trap. What would you suggest? Exception / error log?

Reply:
Exception, because it's a wrong configuration which can be discovered early on.

Reply:
@robertlemke yes, makes sense.
Feel free to fix my friday-afternoon-error-messages:)